### PR TITLE
[Bug fix] Fixing inspect recipes by pinning the mermaid version

### PR DESCRIPTION
**Pull request by sunishsheth2009**
## What changes are proposed in this pull request?
Fixing inspect recipes by pinning the mermaid version

## How is this patch tested?
- [x] Notebook runs and inspects the DAG correctly

## Does this PR change the documentation?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly in the documentation preview.

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [x] Yes. Give a description of this change to be included in the release notes for MLflow users.
Fixing inspect recipes by pinning the mermaid version

### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [x] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [x] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by jinzhang21 — COMMENTED** [review]
What's the background for this change here? Will this work in the long term? It means we loose any improvements from this package in the future, unless we actively look for upgrading it.

**Comment by sunishsheth2009** [comment]
Mermaid upgraded and some breaking changes for us:
image


We currently use internals of mermaid to override styles and few other things. So we need a version that doesn't break. 
Yes that means we need to upgrade manually but it would be better than breaking things. Thoughts?

File: mlflow/recipes/resources/recipe_dag_template.html [hunk 1]
```diff
@@ -134,7 +134,7 @@
     <script type="text/javascript">
       require.config({
         paths: {
-            "mermaid": "https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min",
+            "mermaid": "https://cdn.jsdelivr.net/npm/mermaid@9.3.0/dist/mermaid.min",
```
Comment (jinzhang21) [inline mlflow/recipes/resources/recipe_dag_template.html:137]: I assume this is the latest version that doesn't break our code?

**Review by jinzhang21 — APPROVED** [review]

File: mlflow/recipes/resources/recipe_dag_template.html (hunk 1 above)
Comment (sunishsheth2009) [reply mlflow/recipes/resources/recipe_dag_template.html:137]: Yes correct. This is the 1 before the latest which is 9.4.0

**Pull request merged**